repl/src/main/scala/ammonite/repl/Storage.scala

```diff
@@ -0,0 +1,53 @@
+package ammonite.repl
+
+import acyclic.file
+import java.io.{File, FileInputStream, IOException, FileWriter}
+import org.yaml.snakeyaml.Yaml
+import scala.collection.mutable.ArrayBuffer
+import scala.collection.JavaConversions.asScalaBuffer
+
+object Storage{
```

Comment:
Could we turn this into a class parametrized on the folder path? That feels to me like it could be useful, e.g. if I'm running scripts on a shared machine and want everyone to share the same history/caching